unstoppable-ios-app/domains-manager-ios/Modules/Home/UpdateToWalletGreetingsView/UpdateToWalletGreetingsView.swift

```diff
@@ -74,7 +74,7 @@ private extension UpdateToWalletGreetingsView {
         HStack(spacing: 16) {
             hint.icon
                 .resizable()
-                .squareFrame(24)
+                .squareFrame(40)
                 .foregroundStyle(hint.iconTint)
                 .padding(EdgeInsets(top: 8, leading: 8, bottom: 8, trailing: 8))
```

Comment:
On the mockups the icon's size is 24 pt and there's 8pt padding to each side. It is better to export image in Figma without padding and keep code as it was.